### PR TITLE
Guide alignment

### PR DESCRIPTION
**Pull request by marians**
* Renamed app, component, service and docker image in swarm.json
* Renamed image in Makefile
* Fixed port numbers to 80

### CODE REVIEW

**Comment by marians** [comment]
@advanderveer @luebken Care for a review?

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ docker-run-redis:
 	docker run --name=redis -d redis
 
 docker-run:
-	docker run --link redis:redis -p 8080:80 -ti --rm $(REGISTRY)/$(USERNAME)/$(PROJECT)
```
Comment (luebken) [inline Makefile]: In the past using a high port like 8080 I had less problems on my development machines.

File: swarm.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "app_name": "firstapp",
+  "app_name": "currentweather",
```
Comment (luebken) [inline swarm.json]: Note: We should decide wether we use `currentweather` or `currentweather-app`

**Comment by luebken** [comment]
Minor notes. LGTM

File: swarm.json (hunk 2 above)
Comment (marians) [inline swarm.json]: They are now all on `currentweather`.

**Pull request merged**